src/transformers/models/llama/modeling_llama.py

```diff
@@ -85,11 +83,22 @@ def __init__(self, config: LlamaConfig, device=None):
 
         self.config = config
         if self.rope_type == "default":
-            self.rope_init_fn = _compute_default_rope_parameters
+            base = self.config.rope_theta
```

Comment:
Perfect, the only issue I have is that llama does not do partial! and should always have head dim! 
It's gonna be a little bit of manual work but let's remove what was not there with the model: no partial when model does not do partial, no head dim if model does not have head dim in config!